YoCli/YoCli.csproj

```diff
@@ -20,6 +20,7 @@
     <PackageReference Include="McMaster.Extensions.Hosting.CommandLine" Version="3.0.0" />
     <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="3.1.7" />
     <PackageReference Include="Microsoft.Extensions.Hosting" Version="3.1.7" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
```

Comment:
Can we use Sytem.Text.Json instead of Newtonsoft.Json(2.5MB) keep it lighweight

Reply:
That's good idea !